### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by OlafKozak**
kolejne do kolekcji

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @OlafKozak

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,26 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(lhs == 0 && rhs == 0) return 0;
+    if(lhs < 0) lhs *= -1;
+    if(rhs < 0) rhs *= -1;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: use std::abs

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,26 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(lhs == 0 && rhs == 0) return 0;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:4]: Move return to new line

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
+    while(lhs % divider || rhs % divider){
+        divider--;
+    }
+    return divider;
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(!lhs || !rhs) return 0;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:15]: One command one line

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**